<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <exportmacro>SCHNAPPS_PLUGIN_VOLUME_MODELISATION_API</exportmacro>
 <class>VolumeModelisation_TabWidget</class>
 <widget class="QWidget" name="VolumeModelisation_TabWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>254</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0" colspan="2">
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QComboBox" name="vertexSet_comboBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <item>
      <property name="text">
       <string>- select set -</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>Edges</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_10">
     <property name="text">
      <string>Volumes</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Vertices</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>Faces</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Position</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="position_comboBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <item>
      <property name="text">
       <string>- select attribute -</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="10" column="0" colspan="2">
    <widget class="QComboBox" name="operations_combobox">
     <item>
      <property name="text">
       <string>-select operation-</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="7" column="0" colspan="2">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="13" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="1">
    <widget class="QComboBox" name="edgeSet_comboBox">
     <item>
      <property name="text">
       <string>-select set-</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QComboBox" name="faceSet_comboBox">
     <item>
      <property name="text">
       <string>-select set-</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="dartSet_comboBox">
     <item>
      <property name="text">
       <string>-select set-</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QComboBox" name="volumeSet_comboBox">
     <item>
      <property name="text">
       <string>-select set-</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Darts</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Operations</string>
     </property>
    </widget>
   </item>
   <item row="11" column="0">
    <widget class="QPushButton" name="process_pushButton">
     <property name="text">
      <string>process</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
